src/components/lib/Pages.tsx

```diff
@@ -46,17 +53,19 @@ export abstract class Pages {
    */
   static getResults(props: IStepData, global: IGlobalState): ReactNode {
     const { questionnaire } = global;
-    return (questionnaire.getResults(props.form).map((result) => (
+    return questionnaire.getResults(props.form).map((result) => (
       <li key={`${props.stepId}_${result.id}`} className="padding-bottom-2">
         <span>
           {result.label}:{'  '}
           <b>{result.name}</b>
         </span>
         <div
           className="text-light"
-          dangerouslySetInnerHTML={{ __html: Pages.getReason(props, result, global) }}
+          dangerouslySetInnerHTML={{
```

Comment:
This is interesting to think through. Can someone in the org mistakenly type in some html tags for the reason generate bad html? Something to follow up with in testing.

Reply:
It's a potential risk. The trick is to allow very specific customizations within labels, and this is the way to do that generically.